pkg/network/docker/docker.go

```
@@ -0,0 +1,58 @@
+package docker
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/weaveworks/ignite/pkg/network"
+	"github.com/weaveworks/ignite/pkg/runtime"
+)
+
+const pluginName = "docker-bridge"
+
+type dockerNetworkPlugin struct {
+	runtime runtime.Interface
+}
+
+func GetDockerNetworkPlugin(r runtime.Interface) (network.Plugin, error) {
+	return &dockerNetworkPlugin{r}, nil
```

Does this need to return an error?